dashboard/e2e-test/config/stage.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  host: 'dashboard.stage.mozaws.net',
+  port: 80,
```

Comment:
nit: trailing comma (although technically this isnt a pure JSON file so it's valid).